app/src/main/java/org/holtnet/officialimperialbusiness/ExamActivity.java

```diff
@@ -121,30 +138,37 @@ public void onQuestionsRadioButtonClicked(View view) {
         }
     }
 
+
+    // if you're leveraging the onClick, remove them from the styles so the function name isn't grey below.
```

Comment:
The only reason I don't want to do this is because I'd have to write the words android:onClick with the name like 4x14 times. Would a comment explaining where to find the use of that method be acceptable?

Reply:
Some would say that styles should be in styles.xml, not functionality.  Per the IDE comments, as a result it believes the function isn't used.  Some might say you'd be better off attaching the onClick in your code.  Food for thought.